### PR TITLE
Doc fixes around msq

### PR DESCRIPTION
**Pull request by vogievetsky**
Fixing misc doc typos, changing the example datasets to match the console, editing the known issues page.

### CODE REVIEW

File: docs/multi-stage-query/msq-api.md [hunk 1]
```diff
@@ -652,25 +338,25 @@ print(response.text)
...
+    "location": {
+      "host": "localhost",
+      "port": 8100,
+      "tlsPort": -1
+    },
+    "dataSource": "kttm_simple",
+    "errorMsg": null
+  }
 }
 ```
```
Comment (techdocsmith) [inline docs/multi-stage-query/msq-api.md:362]: should line 382 / 389 be
```
GET /druid/indexer/v1/task/<taskId>/reports
```
Python example uses reports. Report singular 404 for me

**Review by techdocsmith — CHANGES_REQUESTED** [review]
minor nit re: formatting + 2 questions re: syntax

File: docs/multi-stage-query/msq-api.md (hunk 1 above)
Comment (vogievetsky) [reply docs/multi-stage-query/msq-api.md:190]: fixed.

File: docs/multi-stage-query/msq-api.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ Make sure you replace `username`, `password`, `your-instance`, and `port` with t
...
     "context": {
         "maxNumTasks": 3
     }
```
[the remark this answers is not in the document]
Comment (vogievetsky) [reply docs/multi-stage-query/msq-api.md:77]: yup, also missing `}`, fixed.

File: docs/multi-stage-query/msq-api.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ POST /druid/v2/sql/task
...
...json\"}',\n    '[{\"name\": \"added\", \"type\": \"long\"}, {\"name\": \"channel\", \"type\": \"string\"}, {\"name\": \"cityName\", \"type\": \"string\"}, {\"name\": \"comment\", \"type\": \"string\"}, {\"name\": \"commentLength\", \"type\": \"long\"}, {\"name\": \"countryIsoCode\", \"type\": \"string\"}, {\"name\": \"countryName\", \"type\": \"string\"}, {\"name\": \"deleted\", \"type\": \"long\"}, {\"name\": \"delta\", \"type\": \"long\"}, {\"name\": \"deltaBucket\", \"type\": \"string\"}, {\"name\": \"diffUrl\", \"type\": \"string\"}, {\"name\": \"flags\", \"type\": \"string\"}, {\"name\": \"isAnonymous\", \"type\": \"string\"}, {\"name\": \"isMinor\", \"type\": \"string\"}, {\"name\": \"isNew\", \"type\": \"string\"}, {\"name\": \"isRobot\", \"type\": \"string\"}, {\"name\": \"isUnpatrolled\", \"type\": \"string\"}, {\"name\": \"metroCode\", \"type\": \"string\"}, {\"name\": \"namespace\", \"type\": \"string\"}, {\"name\": \"page\", \"type\": \"string\"}, {\"name\": \"regionIsoCode\", \"type\": \"string\"}, {\"name\": \"regionName\", \"type\": \"string\"}, {\"name\": \"timestamp\", \"type\": \"string\"}, {\"name\": \"user\", \"type\": \"string\"}]'\n  )\n)\nPARTITIONED BY DAY",
```
[the remark this answers is not in the document]
Comment (vogievetsky) [reply docs/multi-stage-query/msq-api.md:60]: I think we need a slightly different (and shorter) query for this example

**Review by gianm — APPROVED** [review]
LGTM; looks like @techdocsmith concerns were addressed too, thx @vogievetsky.

(Except for the one about the super-long example query, but that was pre-existing.)

**Pull request merged**